depends/packages/native_qt.mk

```diff
@@ -0,0 +1,77 @@
+# Pretty much taken verbatim from Bitcoin Core
```

Comment:
We should follow [this PR to upgrade Bitcoin Core depends to Qt 5.4.2](https://github.com/bitcoin/bitcoin/pull/6314) and pull it into Armory once it is finalized.